### PR TITLE
Wishlist component

### PR DESCRIPTION
**Pull request by Mohitk391**
Added Wishlist Component.

files to check: 
1) Wishlist.js
2) WishlistContext.js
3) WishlistReducer.js
4) ProductListing.js

preview: https://62d51912f047fa09875816df--elated-mestorf-05ff61.netlify.app/

### CODE REVIEW

File: src/components/VerticalCard/VerticalCard.js [hunk 1]
```diff
@@ -17,9 +20,9 @@ function VerticalCard(props){
                 <div className="card-price-text flex"><p style={{textDecoration: "line-through"}}>Rs. {product.originalPrice}</p> Rs. {product.discountPrice}</div>
                 <div className="card-footer">
                     <div className="card-footer-buttons flex flex-space-evenly">
-                        <button onClick={()=>{
+                        { !itemExistsInCart(cartList, product) ? <button onClick={()=>{
```
Comment (MayankKumar10) [inline src/components/VerticalCard/VerticalCard.js]: make a function try to write it down logic there instead of writing it here

**Review by MayankKumar10 — APPROVED** [review]
Look into this Suggestion everything else Looks Good to me you can merge it, UI Look Good

File: src/pages/authentication/Login.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                        <input type="checkbox" id="remember"/> <label for="remember">Remember me</label>
+                    </div>
+                    <Link  to="/login" class="forgot-pwd">Forgot Password?</Link>
+                </div>
+                <button class="btn btn-hover auth-button sign-in">Login</button>
+                <Link to="/signup"><button class="btn btn-hover auth-button remove-from-wishlist-button">New User? Sign Up</button></Link>
+            </div>
+        </main>
+    </div>
+    );
```
Comment (Ankur9669) [inline src/pages/authentication/Login.js:31]: fix: use className instead of class

File: src/pages/authentication/Login.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+export default function Login() {
+    return (
+        <div class="page-layout">
+        <NavBar />
+        <main class="authenticate-main flex flex-center">
+            <div class="login-body">
+                <div class="title">Login</div>
+                <div class="input-field">
+                    <label class="label" for="email-inp">Email</label>
```
Comment (Ankur9669) [inline src/pages/authentication/Login.js:13]: fix: use htmlFor attribute for accesibility

**Review by Ankur9669 — APPROVED** [review]
Code is looking good

**Pull request merged**